.travis.yml

```diff
@@ -1,17 +1,17 @@
 # Travis configuration for PalMA
 
-sudo: false
-
 language: php
 
+sudo: required
+dist: trusty
```

Comment:
> Isn't `sudo: false` (= container based infrastructure) the recommended standard for new projects?

Yes it is, because the builds start and run faster. The problem is that shellcheck is whitelisted (so could be used with the apt addon and hence in a docker) but not available in the distro, so `sudo apt-get ...` required.